projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Stats Tracking_Pixel
+ *
+ * @package automattic/jetpack-stats
+ */
+
+namespace Automattic\Jetpack\Stats;
+
+use Automattic\Jetpack\Constants;
+use Jetpack_Options;
+
+/**
+ * Stats Tracking_Pixel class.
+ *
+ * Responsible for embedding the Stats tracking pixel.
+ *
+ * @since $$next-version$$
+ */
+class Tracking_Pixel {
+
+	/**
+	 * Stats Build View Data.
+	 *
+	 * @return array.
+	 */
+	public static function build_view_data() {
+		global $wp_the_query;
+
+		$blog     = Jetpack_Options::get_option( 'id' );
+		$tz       = get_option( 'gmt_offset' );
+		$v        = 'ext';
+		$blog_url = wp_parse_url( site_url() );
+		$srv      = $blog_url['host'];
+		$j        = sprintf( '%s:%s', Constants::get_constant( 'JETPACK__API_VERSION' ), Constants::get_constant( 'JETPACK__VERSION' ) );
```

Comment:
Should we leave this here? Maybe we should add this only if the Jetpack plugin is active via a filter